src/components/InvertedFlatList/index.native.js

```diff
@@ -7,6 +8,7 @@ const BaseInvertedFlatListWithRef = forwardRef((props, ref) => (
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         ref={ref}
+        contentContainerStyle={styles.justifyContentEnd}
```

Comment:
We have passed this style to this component in the parent component https://github.com/getusha/App/blob/c97ff978ee258af698628b6a4632074dec451ddb/src/pages/home/report/ReportActionsList.js#L361
What is the reason for duplicating it here?